### PR TITLE
Add missing blank lines after class or function definition.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: searx/engines/google.py [hunk 1]
```diff
@@ -28,6 +28,7 @@
 # based on https://en.wikipedia.org/wiki/List_of_Google_domains and tests
 default_hostname = 'www.google.com'
 
+
```
Comment (asciimoo) [inline searx/engines/google.py]: why is this necessary?

**Comment by stepshal** [comment]
@asciimoo Removed.

File: searx/engines/yandex.py [hunk 2]
```diff
@@ -22,6 +22,8 @@
 language_support = True  # TODO
 
 default_tld = 'com'
+
+
```
Comment (asciimoo) [inline searx/engines/yandex.py]: why?

File: searx/plugins/https_rewrite.py [hunk 3]
```diff
@@ -29,6 +29,7 @@
 description = gettext('Rewrite HTTP links to HTTPS if possible')
 default_on = True
 
+
```
Comment (asciimoo) [inline searx/plugins/https_rewrite.py]: ?

File: searx/plugins/open_results_on_new_tab.py [hunk 4]
```diff
@@ -21,4 +21,5 @@
                       '(JavaScript required)')
 default_on = False
 
+
```
Comment (asciimoo) [inline searx/plugins/open_results_on_new_tab.py]: ?

**Comment by stepshal** [comment]
@asciimoo removed.

File: searx/plugins/search_on_category_select.py [hunk 5]
```diff
@@ -20,4 +20,5 @@
                       'Disable to select multiple categories. (JavaScript required)')
 default_on = True
 
+
```
Comment (asciimoo) [inline searx/plugins/search_on_category_select.py]: ?

File: searx/plugins/vim_hotkeys.py [hunk 6]
```diff
@@ -6,5 +6,6 @@
                       'Press "h" key on main or result page to get help.')
 default_on = False
 
+
```
Comment (asciimoo) [inline searx/plugins/vim_hotkeys.py]: ?

File: searx/webapp.py [hunk 7]
```diff
@@ -85,6 +85,7 @@
 
 default_theme = settings['ui']['default_theme']
 
+
```
Comment (asciimoo) [inline searx/webapp.py]: ?

**Comment by ghost** [comment]
@asciimoo removed.

**Comment by asciimoo** [comment]
thanks
